DQ - Surviving Record/README.md

```diff
@@ -41,6 +51,9 @@ The **DQ - Surviving Record** step allows you to extract the best record (Golden
 
 ## Change Log
 
+* Version 2.0 (25JUL2023)
```

Comment:
You are using a different release date (last code change date) in the About tab of your custom step. Please use the same date in both places.